app/controllers/telegram_webhooks_controller.rb

```diff
@@ -39,24 +24,32 @@ def shows(*args)
         errorMessage = MESSAGE_CHECK_USAGE_COMMAND
       end
 
+      currentMonth = Date.today.strftime("%m").to_s
+
       if errorMessage.nil?
-        @shows = Show.by_day(dayFromArgs)
+        @shows = Show.by_day(dayFromArgs, currentMonth)
       else
         reply_with :message, text: errorMessage
         return
       end
 
+      dateFormated = t(:custom_data_format, :day => dayFromArgs, :month => currentMonth)
+
       if @shows.empty?
-        reply_with :message, text: 'Não há shows programados no dia informado.'
+        reply_with :message, text: t(:empty_shows_results, :dateFormated => dateFormated)
       else
-        response_message = "No dia " + dayFromArgs + ", teremos: \n"
-        
+
+        response_message = t(:shows_title_message, :dateFormated => dateFormated)
+
         @shows.each do |show|
-          response_message += "-> " + show.nome + "\n"
-          response_message += "- Show confirmado: " + (show.confirmado ? "Sim" : "Não") + "\n"
-          response_message += "- Informações da banda: " + show.link_artista + "\n\n"
+          response_message += t(:event_title, :eventTitle => show.name)
+
+          eventConfirmed = show.is_confirmed ? t(:confirmed) : t(:not_confirmed)
+          response_message +=  t(:event_confirmed, :eventConfirmed => eventConfirmed)
+          response_message +=  t(:event_band_info, :moreInfo => show.link_band)
```

Comment:
try to unify the i18n labels nomenclature. +1 for underscore case (more_info) in this case